diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/libmad-0.15.1b/Makefile.am src/libmad-0.15.1b/Makefile.am
--- origsrc/libmad-0.15.1b/Makefile.am	2004-02-16 20:02:03.000000000 -0600
+++ src/libmad-0.15.1b/Makefile.am	2007-02-11 02:50:28.500000000 -0600
@@ -27,6 +27,9 @@
 lib_LTLIBRARIES =	libmad.la
 include_HEADERS =	mad.h
 
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = mad.pc
+
 EXTRA_PROGRAMS =	minimad
 
 minimad_SOURCES =	minimad.c
@@ -101,7 +104,7 @@
 
 version_info =		$(version_current):$(version_revision):$(version_age)
 
-libmad_la_LDFLAGS =	-version-info $(version_info)
+libmad_la_LDFLAGS =	-version-info $(version_info) -no-undefined
 
 mad.h: config.status config.h Makefile.am  \
 		$(srcdir)/COPYRIGHT $(srcdir)/mad.h.sed $(exported_headers)
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/libmad-0.15.1b/configure.ac src/libmad-0.15.1b/configure.ac
--- origsrc/libmad-0.15.1b/configure.ac	2004-01-23 03:41:32.000000000 -0600
+++ src/libmad-0.15.1b/configure.ac	2007-02-11 02:48:58.312500000 -0600
@@ -429,5 +429,6 @@
 dnl AC_SUBST(LTLIBOBJS)
 
 AC_CONFIG_FILES([Makefile msvc++/Makefile  \
+	mad.pc
 	libmad.list])
 AC_OUTPUT
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/libmad-0.15.1b/mad.pc.in src/libmad-0.15.1b/mad.pc.in
--- origsrc/libmad-0.15.1b/mad.pc.in	1969-12-31 18:00:00.000000000 -0600
+++ src/libmad-0.15.1b/mad.pc.in	2007-02-11 02:49:59.046875000 -0600
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: mad
+Description: MPEG Audio Decoder
+Version: @VERSION@
+Libs: -L${libdir} -lmad
+Cflags: -I${includedir}
